sentry_sdk/client.py

```diff
@@ -495,25 +495,27 @@ def capture_event(
         if not is_transaction and not self._should_sample_error(event):
             return None
 
+        tracing_enabled = has_tracing_enabled(self.options)
```

Comment:
Unrelated to the general discussion, `has_tracing_enabled` relies on `enable_tracing` being set, hence we do not solve the immediate need of sending all events via `/envelope` when performance is enabled.